paymentsheet/src/main/java/com/stripe/android/paymentsheet/SavedPaymentMethodMutator.kt

```diff
@@ -110,6 +111,28 @@ internal class SavedPaymentMethodMutator(
                 }
             }
         }
+
+        coroutineScope.launch {
+            currentScreen.collect { currentScreen ->
+                when (currentScreen) {
+                    is PaymentSheetScreen.VerticalMode -> {
+                        // When returning to the vertical mode screen, reset editing to false.
+                        _editing.value = false
+                    }
+                    is PaymentSheetScreen.ManageSavedPaymentMethods,
```

Comment:
Should we make this `else` instead of listing them all out?

Reply:
done!